it/uk/gov/hmrc/gvmshaulierstatus/helpers/CleanMongo.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.gvmshaulierstatus.helpers
+
+import org.mongodb.scala.bson.BsonDocument
+import org.scalatest.{BeforeAndAfterEach, TestSuite}
+import org.scalatestplus.play.BaseOneAppPerSuite
+import play.api.inject.ApplicationLifecycle
+import uk.gov.hmrc.gvmshaulierstatus.repositories.HaulierStatusRepository
+import uk.gov.hmrc.mongo.MongoComponent
+import uk.gov.hmrc.mongo.play.json.PlayMongoRepository
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration.DurationInt
+import scala.concurrent.{Await, Future}
+
+trait CleanMongo extends BeforeAndAfterEach { this: TestSuite with BaseOneAppPerSuite =>
```

Comment:
see if we can reuse the MongoSupport from hmrc-mongo test library, i remember that class almost does the same thing